helm/charts/aleph/templates/exporter.yaml

```diff
@@ -0,0 +1,110 @@
+{{ if .Values.exporter.enabled }}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ .Values.global.namingPrefix }}-exporter
+  labels:
+    app: {{ .Values.global.namingPrefix }}-exporter
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: {{ .Values.global.namingPrefix }}-exporter
+  template:
+    metadata:
+      labels:
+        app: {{ .Values.global.namingPrefix }}-exporter
+      annotations: {{- toYaml .Values.exporter.podAnnotations | nindent 8 }}
+    spec:
+      nodeSelector: {{- toYaml .Values.exporter.nodeSelector | nindent 8 }}
+      restartPolicy: Always
+      securityContext: {{- toYaml .Values.exporter.podSecurityContext | nindent 8 }}
+      containers:
+        - name: {{ .Chart.Name }}
+          image: "{{ .Values.global.image.repository }}:{{ .Values.global.image.tag }}"
+          imagePullPolicy: {{ .Values.global.image.pullPolicy }}
+          command:
+            - "gunicorn"
+            - "--bind"
+            - "0.0.0.0:9100"
+            - "--log-level"
+            - "warn"
+            - "--log-file"
+            - "-"
+            - "aleph.metrics.exporter:app"
+          volumeMounts:
+            - mountPath: /tmp
+              name: tmp-volume
+          securityContext:
+            {{- toYaml .Values.exporter.containerSecurityContext | nindent 12 }}
+          resources:
+            {{- toYaml .Values.exporter.containerResources | nindent 12 }}
+          env:
+            {{- range $key, $value := .Values.global.commonEnv }}
+            - name: {{ $key }}
+              value: {{ $value | quote }}
+            {{- end }}
+            {{- range $key, $value := .Values.global.env }}
+            - name: {{ $key }}
+              value: {{ $value | quote }}
+            {{- end }}
+            - name: ALEPH_DATABASE_URI
+              valueFrom:
+                secretKeyRef:
+                  name: aleph-secrets
+                  key: ALEPH_DATABASE_URI
+            - name: ALEPH_SECRET_KEY
+              valueFrom:
+                secretKeyRef:
+                  name: aleph-secrets
+                  key: ALEPH_SECRET_KEY
+            - name: SENTRY_DSN
+              valueFrom:
+                secretKeyRef:
+                  name: aleph-secrets
+                  key: SENTRY_DSN
```

Comment:
It might make sense to extract these (and possibly other settings that are reused across multiple templates) into a Helm partial. Right now, we duplicate this across the templates for `api`, `ingest-file`, `worker`, and `api`.

Reply:
Good point. I created https://github.com/alephdata/aleph/issues/3496